egs/iam/v1/local/remove_utterances_from_corpus.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env python3
```

Comment:
A cleaned version of this file './local/remove_test_utterances_from_lob.py' is already part of kaldi/master.